examples/with-typescript/pages/pagewithasync.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import 'isomorphic-fetch'
+interface pagesProps {
+  stars: number
+}
+interface pageState {
+  clicked: boolean
+}
+export default class extends React.Component<pagesProps, pageState> {
```

Comment:
i prefer whitespace between every block (blank line after every `}`). give your code some room to breathe man!